### ISSUE TITLE
Can not execute test which parameters contains chrome extension after upgrade to 4.6.12

### ISSUE THREAD

**Issue by ikallala** [serenity-bdd/serenity-core#3086]
Test isn't executed with 4.6.12 if I add extension to chrome options (error: Caused by: net.thucydides.core.webdriver.DriverConfigurationError
Could not instantiate class org.openqa.selenium.chrome.ChromeDriver) 

**extension configuration in serenity.conf**

![image](https://user-images.githubusercontent.com/74077850/224980840-d72cbfe5-cf46-47fc-957d-3da27d05470f.png)

**PS: it worked with 3.4.3 version
PS: if I remove extension configuration when trying to execute test with 4.6.12, it works**

**Comment by wakaleo** [comment]
I can't reproduce this, can you provide a sample project?

**Comment by ikallala** [comment]
any news about this issue ?

**Comment by wakaleo** [comment]
No, unless you have requested commercial support there isn't any SLA on bug fixes for open source projects - however if it is urgent, you are free to look into the code and propose a pull request, or hire someone to do that for you, or request a commercial support package.

**Comment by Darenth** [comment]
@ikallala Hello, did you resolve maybe the problem? I faced the same situation on even the latest version 4.0.29. Currently, it works only on 3.4.3
@wakaleo Hello John, did you try to reproduce it by the provided sample project?

**Comment by wakaleo** [comment]
The sample project looks to be Windows-specific and doesn't run on non-Windows environments - it also uses Serenity 3.x so doesn't reproduce the issue in question. As mentioned above, if you would like help investigating this we can do this as part of a commercial support package.

**Comment by ikallala** [comment]
@Darenth resolved by using another way to add extension
you may add a class to override the newDriver()
![image](https://github.com/serenity-bdd/serenity-core/assets/74077850/0e73aa6a-05a8-4d32-a22f-e3b4a140de5b)
Then in serenity conf, you define the path to this class
![image](https://github.com/serenity-bdd/serenity-core/assets/74077850/5d0fe119-9977-4fa0-93da-9f953a961783)

**Comment by Darenth** [comment]
@ikallala  Thanks a lot for your response, it works!
